packages/compass-aggregations/src/components/pipeline-explain/explain-results.tsx

```diff
@@ -18,7 +18,7 @@ const containerStyles = css({
 });
 
 const cardStyles = css({
-  height: '350px',
+  height: `calc(100vh - 450px)`,
```

Comment:
Where's the 450px number comes from? Is it a calculated value based on some other spacings? Might be worth adding a comment with explanation in case leafygreen changes some spacing of their modal component and we would need to recalculate it or something.

FWIW it feels like maybe something like this belongs more on a modal itself (if that's even possible) not on the card? Feels like what we're trying to do here is for modal container to always as much height as possible (but not more than some `x` value probably so it doesn't stretches too long when people are working with Compass on vertical screens), so something like this for the modal:

```css
.modal {
  height: calc(100vh - var(--leafygreen-modal-vertical-margin));
  max-height: var(--whatever-we-feel-is-appropriate-here-like-600-px-or-something);
}
```

And then the modal content layout should be structured  in a way where card takes as much space as possible?

This feels like something a bit more resilient to changes to the modal content while covering the use case of giving card as much of allowed space.

But also maybe it's a bit of an overkill, I would at least add a comment where the number comes from though 😄